### ISSUE TITLE
Cannot select radio buttons in form

### ISSUE THREAD

**Issue by jnalviano** [pymupdf/PyMuPDF#711]
## Describe the bug (mandatory)
Filling out radio buttons created in another PDF form editor results in the widget `field_value` displaying as `"Yes"` in Python, but no visible change when viewing the PDF in an editor like Adobe Acrobat, Apple Preview or browser.

## To Reproduce (mandatory)

Sample code to reproduce this:

```
with fitz.open(pdf_file) as form:
    page = form[0]
    for field in page.widgets():
        # Select the first radio button only
        if field.field_type_string == 'RadioButton':
            field.field_value = True
            field.update()
            break
    form.save(out_pdf)
```

If I select one of the radio buttons in a PDF editor, save the PDF and then reopen it in PyMuPDF, the `field_value` for that radio button is a numeric index (`0`). Setting `field.field_value = 0` instead of `field.field_value = True` in the above example doesn't work though.

## Expected behavior (optional)
I'd like to be able to use PyMuPDF to select a radio button and have it be visible when viewed in a PDF editor.

## Your configuration (mandatory)
Mac OS X 10.15.7

3.8.2 (default, Sep 24 2020, 19:37:08) 
[Clang 12.0.0 (clang-1200.0.32.21)] 
 darwin 
 
PyMuPDF 1.18.1: Python bindings for the MuPDF 1.18.0 library.
Version date: 2020-10-18 09:50:36.
Built for Python 3.8 on darwin (64-bit).

**Comment by JorjMcKie** [comment]
Sorry, this is currently not supported:
Radio buttons come (usually) in the context of several friends with this common logic: only one of the group can be set on, which sets all the other members to "Off".
I should have documented this somewhere.
I will reclassify to "enhancement" and "documentation bug".

**Comment by JorjMcKie** [comment]
Have to correct my previous post somewhat:
Found a way to set single radio buttons on or off. This will be accepted and correctly displayed by Adobe etc..
What still won't work are mutually dependent radio buttons in a group (where setting one on will automatically set the others off). Every button can always be set on or off w/o implications for the others.
But your requirements should work.
If you are interested in a pre-version, drop me a note please.

**Comment by JorjMcKie** [comment]
Find here is a [pre-version](https://github.com/pymupdf/PyMuPDF-wheels/tree/osx) which should solve your problem.

**Comment by jnalviano** [comment]
Okay awesome, thanks so much for getting that set up so quickly! However, I realized that I gave you the wrong system information. That's my local build, but this needs to run in a docker container on a server. Would it be possible to get the pre-version wheel for the following configuration?
```
3.6.12 (default, Aug 18 2020, 04:20:58) 
[GCC 8.3.0] 
 linux 
 
PyMuPDF 1.18.1: Python bindings for the MuPDF 1.18.0 library.
Version date: 2020-10-18 09:50:36.
Built for Python 3.6 on linux (64-bit).
```

**Comment by jnalviano** [comment]
Cool, this is working beautifully now. Thanks again for the quick update!

**Comment by JorjMcKie** [comment]
New version with these changes being uploaded.

**Comment by dantard** [comment]
Hello, can you kindly tell me if the bug has been solved already? I'm on 1.22.1 and with the following code (very similar to that of @jnalviano) to select the third checkbox in [this](https://royalegroupnyc.com/wp-content/uploads/seating_areas/sample_pdf.pdf) file, for example:
```
    with fitz.open(pdf_file) as form:
       page = form[0]
       rb = 0
    for field in page.widgets():
        # Select the third radio button only
        if field.field_type_string == 'RadioButton':
            if rb == 2:
                field.field_value = True
                field.update()
            rb += 1

    form.save(out_pdf)
```
I get an output file in which, in evince, the third radiobutton is shown as a checkbox and it complains with:
`(evince:1312224): Atk-CRITICAL **: 10:28:27.161: atk_state_set_or_sets: assertion 'ATK_IS_STATE_SET (set)' failed` when I try to select/deselect it.
Thanks in advance
